### ISSUE TITLE
MAUI FlexLayout in a Grid Row of Height Auto will throw exception

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/maui#13004]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/MAUI-FlexLayout-in-a-Grid-Row-of-Height-/10255263)._

---
[severity:It's more difficult to complete my work]
MainPage.xaml
```XAML
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="MauiApp1.MainPage">
	<Grid RowDefinitions="Auto">
		<FlexLayout Wrap="Wrap">
			<Label Text="扩展D"/>
		</FlexLayout>
	</Grid>
</ContentPage>
```
MainPage.xaml.cs
```CSharp
namespace MauiApp1;

public partial class MainPage : ContentPage
{
	public MainPage()
	{
		InitializeComponent();
	}
}
```
Create a MAUI App project, set the MainPage XAML and C# code as above, and Debug on Windows Machine. It will cause a debug break:
![image.png](https://aka.ms/dc/image?name=B6d3dc4d111114ff6a8ab7e49058c2466638096033382014422_20230118-094857-image.png&tid=6d3dc4d111114ff6a8ab7e49058c2466638096033382014422)

---
#### Feedback Bot on 1/17/2023, 06:29 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by jsuarezruiz** [comment]
cc @hartez I think is related with your current work.

**Comment by hartez** [comment]
Yes, this is the same root cause as #11909, so marking it as a duplicate. Will be fixed by #13216

**Comment by hartez** [comment]
Duplicate of #11909
